### DISCUSSION TITLE
How about simulations for Corphins?

### DISCUSSION THREAD

**Discussion by christopherohmer** [JensKrumsieck/PorphyStruct#110] [Ideas]
Hi! 
 
I work on the Ni-cofactor F430, found in methyl-coenzyme m reductase, and unfortunately it is a corphin. I was wondering if a corphin could also be implemented in the simulations. Right now the simulations claim that the corphin is a porphyrin and the simulations are, understandably, rough. 

Thank you so much! 
Chris

**Comment by JensKrumsieck** [comment]
I am aware of this problem as it also exists for corrins. High amounts of saturation of the tetrapyrrol-skeleton leads to high deviation from typical porphyrin non-planar distortion. 
However there is nothing i can do about this as a reference for corphins or corrins definitivly would need more than the 12 references of the extended basis for good fits which would need a complete rewrite of parts of the program. 
As i am currently not working in the porphyrin science anymore i am unable to rewrite parts of PorphyStruct.
All you could use at this place then is its displacement diagrams.

There is a more general structural decomposition program available from Senge group: https://www.kingsbury.id.au/scsd

**Comment by christopherohmer** [comment]
Thanks so much Jens! I figured this would be the case, but it is incredibly
helpful that you took the time to clarify this and I very much appreciate
it! Good luck on your endeavors outside of porphyrins!

Chris

On Sun, Sep 29, 2024 at 8:01 AM Jens Krumsieck ***@***.***>
wrote:

> I am aware of this problem as it also exists for corrins. High amounts of
> saturation of the tetrapyrrol-skeleton leads to high deviation from typical
> porphyrin non-planar distortion.
> However there is nothing i can do about this as a reference for corphins
> or corrins definitivly would need more than the 12 references of the
> extended basis for good fits which would need a complete rewrite of parts
> of the program.
> As i am currently not working in the porphyrin science anymore i am unable
> to rewrite parts of PorphyStruct.
> All you could use at this place then is its displacement diagrams.
>
> There is a more general structural decomposition program available from
> Senge group: https://www.kingsbury.id.au/scsd
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JensKrumsieck/PorphyStruct/discussions/110#discussioncomment-10788683>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BGQCBRZ4E57L3NUHA2NDTHDZY7TXPAVCNFSM6AAAAABO73F2YOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANZYHA3DQMY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Chris Ohmer, PhD
(he/him/his)
Ragsdale Lab
Department of Biological Chemistry
University of Michigan
